schema/config-schema.json

```diff
@@ -135,10 +135,17 @@
                 },
                 "capabilities": {
                     "id": "https://opencontainers.org/schema/bundle/process/linux/capabilities",
-                    "type": "array",
-                    "items": {
-                        "$ref": "defs-linux.json#/definitions/Capability"
-                    }
+                    "type": "object",
+					"properties": {
```

Comment:
Probably want to reformat this to get consistent indents (`make -C schema fmt`).